src/liballoc/vec.rs

```diff
@@ -322,9 +322,9 @@ impl<T> Vec<T> {
     /// ```
     #[inline]
     #[stable(feature = "rust1", since = "1.0.0")]
-    pub fn new() -> Vec<T> {
+    pub const fn new() -> Vec<T> {
```

Comment:
Please mark this with `#[rustc_const_unstable(feature = "const_vec_new")]` so we don't need to insta-stable this.

Reply:
@kennytm Done.